doc/nrf/releases/release-notes-1.5.0.rst

```diff
@@ -0,0 +1,1054 @@
+.. _ncs_release_notes_150:
+
+|NCS| v1.5.0 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
+nRF53 Series devices are now supported for production.
+Bluetooth mesh is supported for development in |release| for prototyping and evaluation.
+Zigbee support for nRF53 Series devices is available for development in |release| for prototyping and evaluation.
+Support for production and deployment in end products is coming soon.
+
+Highlights
+**********
+
+* Introduced support for nRF5340 DK.
+* Added minimal configuration to Bluetooth: Peripheral UART sample for nRF52810 and nRF52811 SoCs.
+* All the models in Bluetooth Mesh Models v1.0.x are now supported for development.
+* Added a basic support for Trusted Firmware-M (TF-M) for nRF5340 and nRF9160.
+* Integrated Apple HomeKit ecosystem support into the |NCS|.
+* Integrated Project Connected Home over IP, which is supported for development.
+* Added production support for nRF5340 for Thread in single protocol configuration, and development support for Thread in multiprotocol operation with Bluetooth LE.
+* Added development support for nRF5340 for Zigbee, including multiprotocol operation with Bluetooth LE.
+* Added nRF9160: Asset Tracker v2 as a fully featured application with optimized power management.
```

Comment:
"Fullly featured" is maybe a bit strong wording. It is still work in progress. Maybe "Added nRF9160: Asset Tracker v2 as a low power real-time configurable application."?

Reply:
Updated.